src/custom/components/swap/TradeSummary/TradeSummaryMod.tsx

```diff
@@ -126,7 +143,11 @@ export default function TradeSummary({
       <RowBetween height={24}>
         <RowFixed>
           <TYPE.black fontSize={12} fontWeight={400} color={theme.text2}>
-            {trade.tradeType === TradeType.EXACT_INPUT ? <Trans>Minimum received</Trans> : <Trans>Maximum sent</Trans>}
+            {trade.tradeType === TradeType.EXACT_INPUT ? (
+              <Trans>Minimum received (incl. fee)</Trans>
```

Comment:
We could abbreviate `Minimum` --> `Min` as that allows for a bit more horizontal space (in the event of bigger numbers etc.).

Reply:
@biocom please make an issue for these and we can address them post merge, thanks!!